apps/barista/src/components/live-example/live-example.ts

```diff
@@ -126,18 +150,49 @@ export class BaLiveExample implements OnDestroy {
     if (this._componentRef) {
       this._componentRef.destroy();
     }
+    this._timerSubscription.unsubscribe();
   }
 
-  /** Whether one of the three sources has been set. */
+  /** @internal Whether one of the three sources has been set. */
   _hasSources(): boolean {
     return Boolean(this.classSource || this.templateSource);
   }
 
+  /** @internal Sets the active tab. */
   _setActiveTab(tab: 'html' | 'ts' | 'scss'): void {
```

Comment:
NIT: you might could write a `type soureType = 'html' | 'ts' | 'scss'` or enum ;)